turborepo-tests/integration/tests/config.t

```diff
@@ -15,7 +15,8 @@ Run test run
     "enabled": true,
     "spacesId": null,
     "ui": false,
-    "packageManager": "npm"
+    "packageManager": "npm",
+    "daemon": null
```

Comment:
Might be good to add a test that confirms the overloading behavior with an env var or flag.

Reply:
thanks! updated in a7216fad72a4bd19d34519d10a29677e9cee3671